angr/engines/ail/engine_light.py

```diff
@@ -311,7 +311,11 @@ def _handle_stmt_ConditionalJump(self, stmt: ailment.statement.ConditionalJump)
         state_false = self.state
         target_true_addr = self._expr_bv(stmt.true_target)
         assert target_true_addr.concrete
-        target_false_addr = self._expr_bv(stmt.false_target)
+        if stmt.false_target is None:
+            # fallthrough
+            target_false_addr = claripy.BVV(self.block.addr + self.block.original_size, self.project.arch.bits)
+        else:
+            target_false_addr = self._expr_bv(stmt.false_target)
```

Comment:
I am conflicted about this, because 1) we don't necessarily know that the continue address should be block.addr + block.size in AIL, for example because the block was duplicated to a new address, and 2) one could conceivably expect that you can put more statements past the one-sided jump. There is logic already which handles if control flow falls through the entire block by picking the graph successor, though it will need some updating to handle when there's more than one out edge.

Reply:
I’ll take it out.